src/Pages/Home/components/TopLeftAlertMenu.tsx

```diff
@@ -5,35 +5,43 @@ import { RightArrowSvg } from "../../../components/Icons/RightArrow"
 import { ListOfAlerts } from "./ListOfAlerts"
 import { ArrowTopSvg } from "../../../components/Icons/ArrowTop"
 import { OpenArrowsSvg } from "../../../components/Icons/OpenArrowsSvg"
+import { MuteAlertSvg } from "../../../components/Icons/MuteAlert"
+import { useTranslation } from "react-i18next"
 
 export function TopLeftAlertMenu() {
+    const {t} = useTranslation();
+    
     const [isAlertOpen, setIsAlertOpen] = useState(false)
+    const [isAlertMuted, setIsAlertMuted] = useState(false)
+    const [unreadAlerts] = useState(0)
     if (isAlertOpen) {
-        return <div className="bg-primary text-primary-foreground w-[25vw] relative">
-            <div className="flex justify-end items-center gap-2 ">
-                <div> Alerts and threats </div>
+        return <div className="bg-[#0d1d3599] shadow-[9px_0px_12.4px_0px_rgba(0,0,0,0.25)] backdrop-blur-[35px] rounded-[20px] text-primary-foreground w-[25vw] relative">
```

Comment:
relative units only and prefer if needed custom shadow across the app then we will use custom class

Reply:
also the color is hard coded

Reply:
Done
See the PR in the internal network.